package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "fastifytest",
+    "version": "1.0.0",
+    "description": "",
+    "main": "app.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "migrate": "node-pg-migrate"
+    },
+    "author": "Ojself",
+    "license": "ISC",
```

Comment:
This will be AGPL as well, but can remove this from the package, this will go in a LICENSE File